examples/gallery/images/contours.py

```diff
@@ -0,0 +1,46 @@
+"""
+Contours
+--------
+The :meth:`pygmt.Figure.contour` method can plot contour lines from a table of points by direct triangulation.
+The data to the triangulation can be provided in one of three options:
```

Comment:
```suggestion
The data for the triangulation can be provided using one of three methods:
```
Optional fix/personal preference, but I think this version reads better.